app/src/controllers/testItem/utils.js

```diff
@@ -28,9 +29,17 @@ export const calculateLevel = (data) =>
 
 export const getQueryNamespace = (levelIndex) => `item${levelIndex}`;
 
-export const getNextPage = (currentLevel, debugMode) => {
-  if (currentLevel === launchLevels.LEVEL_STEP) {
+export const getNextPage = (currentLevel, debugMode, currentType) => {
+  if (
```

Comment:
Can we just do the following:
```
if (currentLevel === launchLevels.LEVEL_STEP) {
  debugMode ? PROJECT_USERDEBUG_LOG_PAGE : PROJECT_LOG_PAGE;
}
```

Reply:
If i understand it clearly, this debugmode working only on the previous level... before this problem.